src/Vehicle/Vehicle.cc

```diff
@@ -1137,6 +1140,27 @@ void Vehicle::_handleGlobalPositionInt(mavlink_message_t& message)
     }
 }
 
+void Vehicle::_handleUtmGlobalPosition(mavlink_message_t& message)
+{
+    mavlink_utm_global_position_t utmGlobalPosition;
+    mavlink_msg_utm_global_position_decode(&message, &utmGlobalPosition);
+
+    // Both values being zero typically corresponds to the current waypoint not being initialized.
+    if (utmGlobalPosition.next_lat == 0 && utmGlobalPosition.next_lon == 0) {
```

Comment:
This case should set the `_positionSetpoint` to something invalid, so that `isValid()` causes the line to disappear

Reply:
Fixed in other PR